### ISSUE TITLE
Document the exact folder for "Keeping the data"

### ISSUE THREAD

**Issue by markiewb** [djmaze/tiddlywiki-docker#9]
I migrated my existing non-docker installation, but it does not work out-of-the-box, because of missing information.

I invoked 
`docker run -d -p 8080:8080 -v /user/john/wiki:/var/lib/tiddlywiki mazzolino/tiddlywiki`

My file structure (with existing tiddlers) was `/user/john/wiki/tiddlers`
But expected was `/user/john/wiki/mywiki/tiddlers`

Please note the hard-coded `mywiki` name from `init-and-run-wiki`

**Comment by kiliankoe** [comment]
Perhaps the wiki/directory name could be stored as an env var as well, making it customizable.

**Comment by djmaze** [comment]
@kiliankoe Good idea. I welcome pull requests as currently I am not actively using this myself.
